<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPatientDialog</class>
 <widget class="QDialog" name="NewPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nowy Pacjent</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Data urodzenia:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameLEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="genderCBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Kobieta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mężczyzna</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>płeć</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwisko:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthDateEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imię:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Płeć:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Proszę podać dane nowego pacjenta:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Numer kartoteki:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileNoLEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Uwagi:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notesTEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="ODPathologyGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Rozpoznanie OD</string>
   </property>
   <widget class="QCheckBox" name="odp_vmt_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>VMT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="odp_undef_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>undefined</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="odp_none_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>brak</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="odp_cavri_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>in CAVRI group</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="odp_other_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>other</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="odp_erm_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ERM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="odp_mh_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MH</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="odp_cnv_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CNV</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="odp_vma_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>VMA</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="OSPathologyGroupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Rozpoznanie OS</string>
   </property>
   <widget class="QRadioButton" name="osp_cavri_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>in CAVRI group</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="osp_erm_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ERM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="osp_vmt_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>VMT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="osp_none_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>brak</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="osp_mh_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MH</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="osp_undef_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>undefined</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="osp_cnv_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CNV</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="osp_other_RButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>other</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="osp_vma_CBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>VMA</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lastNameLEdit</tabstop>
  <tabstop>firstNameLEdit</tabstop>
  <tabstop>birthDateEdit</tabstop>
  <tabstop>genderCBox</tabstop>
  <tabstop>fileNoLEdit</tabstop>
  <tabstop>notesTEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>odp_undef_RButton</tabstop>
  <tabstop>odp_none_RButton</tabstop>
  <tabstop>odp_cavri_RButton</tabstop>
  <tabstop>odp_vma_CBox</tabstop>
  <tabstop>odp_vmt_CBox</tabstop>
  <tabstop>odp_erm_CBox</tabstop>
  <tabstop>odp_mh_CBox</tabstop>
  <tabstop>odp_cnv_CBox</tabstop>
  <tabstop>odp_other_RButton</tabstop>
  <tabstop>osp_undef_RButton</tabstop>
  <tabstop>osp_none_RButton</tabstop>
  <tabstop>osp_cavri_RButton</tabstop>
  <tabstop>osp_vma_CBox</tabstop>
  <tabstop>osp_vmt_CBox</tabstop>
  <tabstop>osp_erm_CBox</tabstop>
  <tabstop>osp_mh_CBox</tabstop>
  <tabstop>osp_cnv_CBox</tabstop>
  <tabstop>osp_other_RButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
